src/components/cart-list/index.tsx

```diff
@@ -4,7 +4,22 @@ import { List } from '../general/list';
 import { CartContext } from '../../contexts/Cart';
 
 export const CartList: React.FC = () => {
-  const { cart } = useContext(CartContext);
+  const { cart, setCart } = useContext(CartContext);
+  const changeQuantity = (id: string, amount: number) => {
+    setCart((currCart) => {
+      return currCart.map((currItem) => {
+        if (currItem.id === id) {
+          const newQty =
+            currItem.quantity + amount > 0 ? currItem.quantity + amount : 0;
+          return { ...currItem, quantity: newQty };
+        }
+        return { ...currItem };
+      });
+    });
+  };
+  const removeItem = (id: string) => {
```

Comment:
fab!! 😁 

up to you if you want to create a utils folder and put the functions in there like Erica mentioned earlier?